platform/x11/os_x11.cpp

```diff
@@ -3428,6 +3428,7 @@ Error OS_X11::move_to_trash(const String &p_path) {
 	// The trash can is successfully created, now move the given resource to it.
 	// Do not use DirAccess:rename() because it can't move files across multiple mountpoints.
 	List<String> mv_args;
+  mv_args.push_back("--backup=t");
```

Comment:
`backup` is no valid argument for the BSD / macOS flavor of `mv`.

```suggestion
#if defined(__FreeBSD__) || defined(__OpenBSD__) || defined(__NetBSD__)
  // Something else
#else
  mv_args.push_back("--backup=t");
#endif
```

Reply:
And indentation should use tabs not spaces.

Reply:
Thank you so much for your review! Unfortunately, it looks like I can't test this on the master branch. It looks like my graphics card on my Linux machine doesn't satisfy the Vulkan version. Is there a way to suppress this error just so I can test exporting?

Reply:
There's no headless DisplayServer yet, so I'm afraid that's not possible. If you use a recent Mesa version, you could try installing Zink (which is a software Vulkan implementation) but it will be *very* slow.

You could also install and use [SwiftShader](https://github.com/google/swiftshader), although it won't render Godot correctly – it'll just display a black screen and spam error messages.

Reply:
You can get SwiftShader working (at least 2D/GUI rendering) with few small patches to it and Godot, see https://github.com/bruvzg/bruvzg.github.io/tree/master/godot_swiftshader

Also make sure you are using `debug` build, `release` and `release_debug` builds do not work correctly on many cards.

Reply:
Okay awesome, thanks for the tips. I'll give this a shot.

Reply:
@Calinou note that Zink is a Vulkan implementation of OpenGL, not a software backend.

Reply:
@akien-mga I confused Zink and Lavapipe (formerly known as Vallium), sorry.

Reply:
I had a few issues with installing SwiftShader, but I seem to have it working now. How do I go about using it to run Godot?

Reply:
Added swiftshader/build/Linux to rpath of executable as explained in SwiftShader readme. Unfortunately, still seems to not be working. Any advice?

Reply:
You should run it with `VK_ICD_FILENAMES` env variable set to `json` config file.

e.g `VK_ICD_FILENAMES=~/swiftshader.json godot.linuxbsd.tools.64`

`json` config file is something like:
```json
{
    "file_format_version" : "1.0.0",
    "ICD": {
        "library_path": "FULK_PATH_TO/libvk_swiftshader.so",
        "api_version" : "1.0.0"
    }
}
```

Reply:
Great news! It worked, except only displayed a black screen. I did do the changes that you have here
https://github.com/bruvzg/bruvzg.github.io/tree/master/godot_swiftshader
Possibly time for me to get a newer computer...